Corona-Warn-App/src/main/java/de/rki/coronawarnapp/ui/view/TestResultSectionView.kt

```diff
@@ -26,30 +27,32 @@ constructor(
     attrs: AttributeSet? = null,
     defStyleAttr: Int = 0
 ) : ConstraintLayout(context, attrs, defStyleAttr) {
+    private val binding = ViewTestResultSectionBinding.inflate(LayoutInflater.from(context), this)
```

Comment:
I'm afraid that `inflate` is inflating another view that is not attached to the current layout. Please use `bind(this)`
instead.

Reply:
`CircleProgress` for example is using `inflate` too. But I can change it later

Reply:
Done